httpie/output/processing.py

```
@@ -0,0 +1,50 @@
+
+    def get_converter(self, mime):
+        if is_valid_mime(mime):
+            for converter_class in plugin_manager.get_converters():
+                if converter_class.supports(mime):
+                    return converter_class(mime)
+
+
+class Formatting(object):
+    """A delegate class that invokes the actual processors."""
+
+    def __init__(self, groups, env=Environment(), **kwargs):
```

Ruff (0.8.2)

Do not perform function call `Environment` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)